tools/hardware/outputs/c_header.py

```diff
@@ -27,7 +27,7 @@
 
 #pragma once
 
-#define physBase {{ "0x{:x}".format(physBase) }}
+#define physBase_raw {{ "0x{:x}".format(physBase) }}
```

Comment:
shall we make it uppercase, as the usual macros?

Reply:
Yes, that's a good idea.

Reply:
- [ ] rename to PHYS_BASE_RAW